app/Http/Controllers/TestResultController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use App\Test;
+use App\Result;
+use Illuminate\Http\Request;
+use App\Http\Resources\TestResult;
+use App\Http\Resources\TesteeTestResult;
+use App\Http\Resources\Result as ResultDetails;
+
+class TestResultController extends Controller
+{
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function getTestResultById($id)
```

Comment:
Why did you not kept the controller methods naming principals?
Have look at the whole project you have now.
So, to get all the resources we have **index** method. To get details of any specific resource we have **show** method, etc.

Reply:
Also, change the parameter name from $id to $testId. I tough its test result ID.